test/unit/http/common/backoff_test.dart

```diff
@@ -0,0 +1,184 @@
+// Copyright 2015 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('browser')
+import 'dart:async';
+import 'dart:html';
```

Comment:
I don't think this has any dependency on `dart:html`, does it? If not, should remove this import and change the `@TestOn` annotation to `@TestOn('vm || browser');`